bambam.py

```diff
@@ -189,7 +189,12 @@ def print_letter(char):
 pygame.init()
 
 # figure out the install base to use with image and sound loading
-progInstallBase = os.path.dirname(os.path.realpath(sys.argv[0]));
+dataDirs = [os.path.join(os.path.dirname(os.path.realpath(sys.argv[0])),'data')];
```

Comment:
Can you please keep the ``progInstallBase`` variable around?
I patch it in the Debian package and this will make my life easier.
Also, missing space after comma.